### ISSUE TITLE
Utilize Prefer Header

### ISSUE THREAD

**Issue by ibnesayeed** [oduwsdl/memgator#94]
As proposed in http://ws-dl.blogspot.com/2016/08/2016-08-15-mementos-in-raw-take-two.html it should be possible to request for the original content for the proxy without using the `id_` hack. However, it is not reliable yet until many archives start supporting that.

**Comment by machawk1** [comment]
Do any of the archives that are aggregated by default support this, @ibnesayeed? I think support for something like this was added to pywb, so we might look to those to identify support.

**Comment by ibnesayeed** [comment]
Yes, we need to asses the support of `Prefer` header in replay systems and their deployment base before we consider putting any cycles in coding this.

**Comment by machawk1** [comment]
@ikreymer Do you have a running list of publicly available archives that are using pywb and if any have Prefer mod enabled?
